packages/react-native/Libraries/AppDelegate/RCTAppDelegate.h

```diff
@@ -127,13 +81,6 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)setRootView:(UIView *)rootView toRootViewController:(UIViewController *)rootViewController;
 
-/// This method controls whether the App will use RuntimeScheduler. Only applicable in the legacy architecture.
-///
-/// @return: `YES` to use RuntimeScheduler, `NO` to use JavaScript scheduler. The default value is `YES`.
-- (BOOL)runtimeSchedulerEnabled;
```

Comment:
I'm not sure if this is needed. It wasn't used anywhere in the code

Reply:
we can probably remove this and hardcode it to true. There is no reason to have it to false